packages/app_ui/lib/src/widgets/block_dialog.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:app_ui/app_ui.dart';
+import 'package:flutter/material.dart';
+
+class BlockDialog extends StatelessWidget {
+  const BlockDialog({
+    required this.title,
+    required this.btnText,
+    required this.content,
+    required this.onPressed,
+    super.key,
+  });
+  final String title;
+  final String content;
+  final String btnText;
+  final Function onPressed;
```

Comment:
this should be a final void Function() I think, or a VoidCallback which is a typedef very used for simplicity in the syntax. Let's avoid ignoring the rules and try to comply whenever possible. Must of the time they are indicators of things that we could do better. You probably will have to add an import to make the suggestion work
```suggestion
  final VoidCallback onPressed;